autoload/gitgutter/utility.vim

```diff
@@ -176,15 +176,20 @@ endfunction
 
 function! s:use_known_shell() abort
   if has('unix') && &shell !=# 'sh'
-    let [s:shell, s:shellcmdflag, s:shellredir] = [&shell, &shellcmdflag, &shellredir]
+    let [s:shell, s:shellcmdflag, s:shellredir, s:shellpipe, s:shellquote, s:shellxquote] = [&shell, &shellcmdflag, &shellredir, &shellpipe, &shellquote, &shellxquote]
     let &shell = 'sh'
     set shellcmdflag=-c shellredir=>%s\ 2>&1
   endif
+  if has('win32') && (&shell ==# 'pwsh' || &shell ==# 'powershell')
+    let [s:shell, s:shellcmdflag, s:shellredir, s:shellpipe, s:shellquote, s:shellxquote] = [&shell, &shellcmdflag, &shellredir, &shellpipe, &shellquote, &shellxquote]
+    let &shell = 'cmd.exe'
+    set shellcmdflag=/s\ /c shellredir=>%s\ 2>&1 shellpipe=>%s\ 2>&1 shellquote= shellxquote="
```

Comment:
The `shellxquote` value appears to be a double-quotation mark.  Is that right?  I would have expected `'"'` or `"\""` in that case.

Reply:
Here is my default:
![shellxquote](https://user-images.githubusercontent.com/45818681/101926194-27fd9300-3bd3-11eb-92a9-81d872c8163b.png)

Reply:
Ah, I guess option values set with `:set` don't need quoting.